packages/app/src/server/models/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   // PageArchive: require('./page-archive'),
   PageTagRelation: require('./page-tag-relation'),
   User: require('./user'),
+  UserRegistrationOrder: require('./user-registration-order'),
```

Comment:
Please do not set the UserRegistrationOrder model here since setting models here is an old method

To define a model, using getOrCreateModel and importing it from other files is better now.